File: src/test/java/no/nav/pensjon/innsyn/domain/source/HardcodedFppAfp.java
package no.nav.pensjon.innsyn.domain.source;

import no.nav.pensjon.innsyn.domain.FppAfp;
import no.nav.pensjon.innsyn.source.EntityGetter;

import java.util.List;

public class HardcodedFppAfp implements EntityGetter<FppAfp> {

    @Override
    public List<FppAfp> getEntities() {
        return List.of(
                new FppAfp(
                        "status",
                        123456.78d,
                        "gjelderFom",
                        "gjelderTom",
                        123456.78d,
                        "afpType"));
    }
}

File: src/test/kotlin/no/nav/pensjon/innsyn/domain/source/HardcodedFppAfp.kt
package no.nav.pensjon.innsyn.domain.source

import no.nav.pensjon.innsyn.domain.FppAfp

class HardcodedFppAfp : EntityGetter<FppAfp?> {
    override val entities = listOf(
                FppAfp(
                        "status",
                        123456.78,
                        "gjelderFom",
                        "gjelderTom",
                        123456.78,
                        "afpType"))
}